source/jormungandr/readme.md

```diff
@@ -0,0 +1,105 @@
+
+# Jormungandr
+Web service around the c++ core(s). This is the navitia api front end, based on flask.
+
+:snake: **python module**
+
+## Norse mythology
+In the Norse mythology, Jörmungandr is a sea serpent that grew so large that it can surround the earth, hence his name: Midgard Serpent. He will be killed by Thor during Ragnarok.
+
+# Setup
+## VirtualEnv
+
+```sh
+cd navitia/source/jormungandr
+mkvirtualenv jormungandr
+pip install -r requirements.txt -r requirements_dev.txt -U
+```
+
+# Configure
+
+Jormungandr uses 2 different config files:
+* `jormungandr.json` : that describes the Kraken's instances and coverage
+* `custom_settings.py` : a python script that overwrites default settings
+
+## jormungandr.json
+
+Create a file `jormungandr.json` in a custom directory like:
+
+```json
+{
+    "key": "some_region",
+    "socket": "ipc:///tmp/default_kraken"
+}
+```
+
+You can copy the one from [documentation/examples/config/Jormungandr.json](https://github.com/CanalTP/navitia/blob/dev/documentation/examples/config/Jormungandr.json)
+
+Make sure `key` and `socket` respectively match `instance_name` and `zmp_socket` from your Kraken config file ([kraken.ini](https://github.com/CanalTP/navitia/blob/dev/documentation/examples/config/kraken.ini)).
+
+## custom_settings.py [optional]
+
+This helps you overwrite the default settings if needed (from [default_settings.py](https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/default_settings.py)).
+
+# Run 
+
+To run the web service, you'll need to:
+
+* set `JORMUNGANDR_INSTANCES_DIR` to point at the directory that contains your `jormungandr.json`.
+* add `navitia/source/jormungandr` and `navitia/navitiacommon` to your python path via `PYTHONPATH`.
+* [optionaly] set `JORMUNGANDR_CONFIG_FILE` to point at your `custom_settings.py`.  
```

Comment:
Mixing custom `settings.py` and configuration by environnement variable might not be the best idea, I would recommend using one or the other.
Typically I only use env var for my local jormungandr.

Reply:
I've updated the doc mentioning that we prefer envvars over custom settings (ie. 
e9d7de9)